Form1.cs

```diff
@@ -6150,6 +6162,7 @@ private async void actionTimer_TickAsync ( object sender , EventArgs e )
 
                                    if ( Form2.config.Perpetuance && Form2.config.regenPerpetuance && currentSCHCharges > 0 && HasAbility ( "Perpetuance" ) && !plStatusCheck ( StatusEffect.Perpetuance ) )
                                    {
+                                        JobAbility_Wait("Regen, Perpetuance", "Perpetuance");

```

Comment:
This line I just copied from another spell's entry. For whatever reason, it just didn't exist.